eng/pipelines/runtimelab-official.yml

```diff
@@ -0,0 +1,60 @@
+trigger:
+  batch: true
+  branches:
+    include:
+    - feature/*
+  paths:
+    include:
+    - '*'
+    exclude:
+    - '**.md'
+    - eng/Version.Details.xml
+    - .devcontainer/*
+    - .github/*
+    - docs/*
+    - LICENSE.TXT
+    - PATENTS.TXT
+    - THIRD-PARTY-NOTICES.TXT
+
+variables:
+- template: /eng/pipelines/common/variables.yml
+  parameters:
+    templatePath: 'templates-official'
+
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), ne(variables['Build.Reason'], 'PullRequest')) }}:
+  - name: TeamName
+    value: dotnet-core
+extends:
+  template: /eng/pipelines/common/templates/pipeline-with-resources.yml@self
+  parameters:
+    isOfficialBuild: true
```

Comment:
Below in the file, there are places that check `isOfficialBuild` instead of hardcoding it. What is the intent behind that?

Reply:
It is likely that some are auto-generated from the conversion script, and some were just written by me by hand. Thus you have me hardcoding things to true (because this is the official build) and the script not knowing what to do and pulling the variable.